<?xml version="1.0"?>
<launch>
<!--     <remap from="img_topic_subscribed" to="cam0/image_raw" /> -->
<!--     <remap from="pattern_topic_published" to="cam0/calib_pattern_recognised" /> -->
    <node pkg="pp_alvium_driver" name="ros_alvium_nir" type="ros_alvium_nir.py" output="screen">
<!--         <rosparam file="$(find pp_alvium_driver)/config/params_nir.yaml" command="load"/> -->
<!--         <remap from="pp/rgb_raw" to="pp/nir_raw"/> -->
        <rosparam>
        path2cam_calib_yaml: /home/maciej/ros1_wss/pp_collector/src/pp_alvium_driver/calib/210722_nir.yaml
        camera_id: DEV_1AB22C00C28B
        frequency: 60
<!--         2200 in max exposure time for both indoors and outdoors                 -->
        max_exposure_time: 2200
        publish_camera_info: True
        published_caminfo_topic: /pp/nir_cam_info
        published_img_topic: /pp/nir_raw
        </rosparam>
    </node>

</launch>
